.changeset/curvy-frogs-return.md

```diff
@@ -0,0 +1,5 @@
+---
+'@builder.io/qwik': patch
+---
+
+Introducing the `unstableFeatures[]` option to the Vite plugin. This allows you to opt in to features that are not guaranteed to have a stable API.
```

Comment:
IMHO `experimentalFeatures` sounds better to me.

Reply:
yeah, for me `experimentalFeatures` is better too